README.md

```diff
@@ -36,7 +36,31 @@ Note: the `--destroy-storage` will delete any data persisted by MySQL.
 
 ## Relations
 
-There are no relations implemented yet.
+We have added support for two legacy relations (from the [mysql-innodb-cluster](https://charmhub.io/mysql-innodb-cluster) charm):
+
+1. `db-router` is a relation that one uses with the [mysql router](https://charmhub.io/mysql-router) charm. The following commands can be executed to deploy and relate to the keystone charm:
+```shell
+# Deploy the relevant charms
+juju deploy -n 3 ./mysql_ubuntu-20.04-amd64.charm mysql
```

Comment:
Once this is deployed, you might want to change this so it points to the charmhub URL. For now, you may also need to say to `pack` it first

Reply:
Good call, updated README in [5bd1002](https://github.com/canonical/mysql-operator/pull/17/commits/5bd10022d7894cd2fbd2e4cf18d27470a3d3181e)